flent/plotters.py

```diff
@@ -1228,7 +1229,10 @@ def _do_scaling(self, axis, data, btm, top, unit=None, allow_log=True):
             axis.set_ylim(0, top_scale)
         else:
             if self.log_base:
-                axis.set_yscale('log', basey=self.log_base)
+                if MPL_VER >= LooseVersion("3.2"):
```

Comment:
You have an off-by-one error here; the `base` argument was introduced in matplotlib 3.3: https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.3.0.html#log-symlog-scale-base-ticks-and-nonpos-specification

Which kinda betrays that you probably didn't test this on old matplotlib versions? Please do so :)

Reply:
Ahh, yes. I must admit that I went through the matplotlib git repo, found the commit where it changed, and made an off-by-one error. I did not test this on an old matplotlib version at all. I will do so in the future.